src/frontend/js/types/commonDataProps.ts

```diff
@@ -6,8 +6,10 @@ import { Nullable } from 'utils/types';
  */
 export interface CommonDataProps {
   context: {
+    allowed_hosts: string[];
```

Comment:
I don't find where you are using it

Reply:
It's a mistake. I forgot to rename `allowed_hosts` into `redirect_whitelist`